<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fish</class>
 <widget class="QMainWindow" name="Fish">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(67, 67, 67);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>330</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Food</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>311</width>
       <height>151</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButtonVegetarian">
        <property name="text">
         <string>Vegetarian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonMeatEater">
        <property name="text">
         <string>Meateater</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonOmnivorous">
        <property name="text">
         <string>Omnivorous</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>320</width>
      <height>140</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>141</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>skelet</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButtonBony">
        <property name="text">
         <string>Bony</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonCartilaginous">
        <property name="text">
         <string>Cartilaginous</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>630</y>
      <width>195</width>
      <height>37</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButtonSave">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(3, 218, 197);
font: 75 14pt &quot;Arial&quot;;
color: rgb(15, 15, 15);</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBack">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(3, 218, 197);
font: 75 14pt &quot;Arial&quot;;
color: rgb(15, 15, 15);</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(64, 64, 64);
font: 75 18pt &quot;Arial&quot;;
background-color: rgb(187, 134, 252);</string>
    </property>
    <property name="text">
     <string> Fish </string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>320</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Jaw</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButtonwithJaw">
        <property name="text">
         <string>With the jaw</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonWithoutJaw">
        <property name="text">
         <string>Without jaw</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEditDescription">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>370</y>
      <width>421</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;Arial&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="placeholderText">
     <string>Description...</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>401</width>
      <height>166</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
color: rgb(64, 64, 64);
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string> animal name : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditName">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
color: rgb(64, 64, 64);
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string> animal nike name : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditNikeName">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
color: rgb(64, 64, 64);
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string> animal age : </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditAge">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
color: rgb(64, 64, 64);
background-color: rgb(187, 134, 252);</string>
         </property>
         <property name="text">
          <string> number of fins  :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxFins">
         <property name="styleSheet">
          <string notr="true">font: 75 12pt &quot;Arial&quot;;
background-color: rgb(220, 220, 220);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>320</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 90, 90);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>Sex</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>111</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButtonMale">
        <property name="text">
         <string>Male</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonFemale">
        <property name="text">
         <string>Female</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
